platform/ui/src/components/studyBrowser/Thumbnail.js

```diff
@@ -22,28 +40,73 @@ function ThumbnailFooter({
       </div>
     );
   };
+
+  const getWarningContent = (warningList) => {
+    if (Array.isArray(warningList)) {
+      const listedWarnings = warningList.map((warn, index) => {
+        return <li key={index}>{warn}</li>;
+      });
+
+      return <ol>{listedWarnings}</ol>;
+    } else {
+      return <React.Fragment>{warningList}</React.Fragment>;
+    }
+  };
+
+  const getWarningInfo = (SeriesNumber, warningList) => {
+      return(
+        <React.Fragment>
+        {warningList.length != 0 ? (
+          <OverlayTrigger
+            key={SeriesNumber}
```

Comment:
Looks like a  new component was born, maybe we should extract it? and use here like <WarningInfo />?

Reply:
I have not understood this comment. Do you mean to move the react element in anther file/function?

Reply:
Fro line 56 to 85 is a new react component specifically to display inconsistencies which could probably live outside the thumbnail and just used there like import { WarningTooltip } or something, not necessary though :)